opendatahub-io/odh-dashboard
# [RHOAIENG-6573] Model Detail - Registered Deployments Section

https://issues.redhat.com/browse/RHOAIENG-6573

## Description
- Added labels to creation of model deployments
- Added registered deployments to model version details and archived model version details.

<img width="1431" alt="image" src="https://github.com/user-attachments/assets/04d2bba6-18f4-42f1-8795-a2d7d5814fa6">

(cc @yih-wang)

## How to test
Navigate to the version details of a given model version (archived or otherwise). Click on the registered deployments tab, verify the contents match the design in terms of the alert, table columns, and table data. If no table data exists, that means the version has no registered deployments, so you can create one using the toolbar action "Deploy".

## Test Impact
Added unit tests and a cypress test verifying data can render in the registered deployments table.

## Request review criteria:
Self checklist (all need to be checked):

If you have UI changes: 

After the PR is posted & before it merges:
